napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -127,6 +127,9 @@ def _on_enabled_checkbox(self, state: int):
                 )
                 return
 
+    def _warn_pypi_install(self):
+        return running_as_constructor_app()
```

Comment:
```suggestion
        return running_as_constructor_app() or is_conda_package('napari') 
```
I think it could be worth showing this for napari-installed-from-conda environments too?
The same risk exists--I've broken envs this way before!

Reply:
That makes sense to me :+1: